src/v2/Apps/Consign/Components/InDemandNow/ConsignTopArtists.tsx

```diff
@@ -85,11 +90,48 @@ const TopArtists: React.FC<ConsignTopArtistsQuery["response"]> = props => {
     return null
   }
 
+  // Iterate over microfunnel artworks and a) group into "chunks" of four for
+  // displaying in rows; b) average each artwork realized price; c) shuffle the
+  // display so that its a bit less static.
   const recentlySoldArtworks = chunk(
     shuffle(
-      microfunnelItems.map(
-        artwork => artwork?.artworksConnection?.edges?.[0]?.node
-      )
+      microfunnelItems.map(artwork => {
+        if (artwork?.artworksConnection?.edges?.length === 0) {
+          return null
+        }
+
+        const { edges } = artwork.artworksConnection
+
+        // Get realized price average for artworks
+        const realizedPriceAverage = edges.reduce(
+          (total, artwork, index, artworks) => {
+            // Done adding up realized prices, now average
+            if (index === artworks.length - 1) {
+              const averageInDollars = formatCentsToDollars(
+                (total / artworks.length) * 100
+              )
+              return averageInDollars
+            }
+
+            // Value comes back from MP formatted; strip dollars and commas
+            const realizedPrice = artwork.node.realizedPrice
+              .replace("$", "")
+              .replace(",", "")
+
+            // Sum up values
+            return total + Number(realizedPrice)
+          },
+          0
+        )
```

Comment:
I think this can be simplified like this:

```suggestion
        const realizedPriceAverage = formatCentsToDollars(100 * edges.reduce(
          (total, artwork) => {
            // Value comes back from MP formatted; strip dollars and commas
            const realizedPrice = artwork.node.realizedPrice
              .replace("$", "")
              .replace(",", "")
            return (total + realizedPrice) / artworks.length
          },
          0
        ))
```

Reply:
I like this very much 👍